src/settings-ui/Settings.UI/SettingsXAML/Views/MouseUtilsPage.xaml

```diff
@@ -196,7 +196,7 @@
                                 <NumberBox
                                     MinWidth="{StaticResource SettingActionControlMinWidth}"
                                     LargeChange="100"
-                                    Minimum="100"
+                                    Minimum="0"
```

Comment:
Why not setting this to 1 instead of doing a hack?

Reply:
I figured that would lead to an annoying interaction.
If you try to go to 0 and end up on 1 and then increase the duration again using the arrows, you'd end up on 11 or 101.
But that's probably worth it to avoid the hack, right?

Reply:
There are two xaml properties for this control you can use to manipulate this behavior: `...SmallChange` and `...LargeChange`. Not sure if it makes sense to adjust them.

Reply:
That's my thinking. Then I'd have to add conditionals for those. Which in turn could lead to users getting annoyed because it doesn't work "as it should".
That's my justification for my hack. If we don't want that, then I think I'd prefer to just set the limit to 1 as you suggested.